server/routes.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const api = require('./api.js');
+
+const filePaths = {
+    index: path.resolve(__dirname, '../build/index.html'),
+    admin: path.resolve(__dirname, '../build/admin.html'),
+    assets: path.resolve(__dirname, '../build'),
+    csvFile: path.resolve(__dirname, '../build/registered.csv')
+};
+
+module.exports = [
+    {
+        method: 'GET',
+        path: '/',
+        handler: {
+            file: filePaths.index
+        }
+    },
+    {
+        method: 'GET',
+        path: '/{fileName*}',
+        handler: {
+            directory: {
+                path: filePaths.assets
+            }
+        }
+    },
+    {
+        method: 'POST',
+        path: '/register',
+        handler: api.saveFormData
+    },
+    {
+        method: 'GET',
+        path: '/admin',
+        config: {
+            auth: 'simple',
+            handler: {
+                file: filePaths.admin
+            }
+        }
+    },
+    {
+        method: 'GET',
+        path: '/registeredUsers',
+        config: {
+            auth: 'simple',
+            handler: api.getRegisteredUsers
+        }
+    },
+    {
+        method: 'GET',
+        path: '/download',
+        handler: api.download
+    }
```

Comment:
Is the `download` function going to be replaced by the `getRegisteredUsers` function at some point? If not, it should probably include authorisation as well